CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 - Use GitHub tags (via `git ls-remote`) to determine the latest Tuist version when installing/updating Tuist [#3985](https://github.com/tuist/tuist/pull/3985) by [@ezraberch](https://github.com/ezraberch)
 
+- Remove default MacCatalyst support when framework deployment target is set to iOS and/or iPad [#4134](https://github.com/tuist/tuist/pull/4134) by [@TheInkedEngineer](https://github.com/TheInkedEngineer)  
```

Comment:
Not sure why I see this old changelog here 🤔 
For example, if you see on main, there should be many more entries in Next, and last release should be 2.7.2 🤷‍♂️

Reply:
I can just rebase